### PR TITLE
dev, v3.0, v2.1: add DM upgrade advice

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? 
Add incompatible DM upgrade version advice after this [PR](https://github.com/pingcap/dm/pull/212) is merged.

### What is the related PR or file link(s)? 
https://github.com/pingcap/dm/pull/212

### Checklist 

- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 1]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+### 主要变更
+
+从此版本开始，所有的配置进行严格的检查，遇到不识别的配置会报错，这样的目的是为了确保用户始终准确的了解自己的配置。
+
+### 升级操作示例
+
+启动 dm-master/dm-worker 前确保废弃的配置信息已经删除，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
+可能遗留的废弃配置:
+
+- dm-worker.toml中的 `meta-file`
```
Comment (amyangfei) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- dm-worker.toml 中的 `meta-file`
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 2]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+- task.yaml中的 `mysql-instances` 中的 `server-id`
```
Comment (amyangfei) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- task.yaml 中的 `mysql-instances` 中的 `server-id`
```

File: dev/reference/tools/data-migration/dm-upgrade.md (hunk 1 above)
Comment (3pointer) [reply dev/reference/tools/data-migration/dm-upgrade.md]: Done

File: dev/reference/tools/data-migration/dm-upgrade.md (hunk 2 above)
Comment (3pointer) [reply dev/reference/tools/data-migration/dm-upgrade.md]: Done

**Comment by amyangfei** [comment]
PTAL @CaitinChen

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 3]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+Release Version: v1.0.0-rc.1-12-gaa39ff9
+Git Commit Hash: aa39ff981dfb3e8c0fa4180127246b253604cc34
+Git Branch: dm-master
+UTC Build Time: 2019-07-24 02:26:08
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，所有的配置进行严格的检查，遇到不识别的配置会报错，这样的目的是为了确保用户始终准确的了解自己的配置。
```
Comment (lilin90) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 4]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+
+### 升级操作示例
+
+启动 dm-master/dm-worker 前确保废弃的配置信息已经删除，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

File: v2.1/reference/tools/data-migration/dm-upgrade.md [hunk 5]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+Release Version: v1.0.0-rc.1-12-gaa39ff9
+Git Commit Hash: aa39ff981dfb3e8c0fa4180127246b253604cc34
+Git Branch: dm-master
+UTC Build Time: 2019-07-24 02:26:08
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，所有的配置进行严格的检查，遇到不识别的配置会报错，这样的目的是为了确保用户始终准确的了解自己的配置。
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
```

File: v3.0/reference/tools/data-migration/dm-upgrade.md [hunk 6]
```diff
@@ -20,6 +20,30 @@ aliases: ['/docs-cn/tools/dm/dm-upgrade/']
...
+Release Version: v1.0.0-rc.1-12-gaa39ff9
+Git Commit Hash: aa39ff981dfb3e8c0fa4180127246b253604cc34
+Git Branch: dm-master
+UTC Build Time: 2019-07-24 02:26:08
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，所有的配置进行严格的检查，遇到不识别的配置会报错，这样的目的是为了确保用户始终准确的了解自己的配置。
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
```

File: v2.1/reference/tools/data-migration/dm-upgrade.md [hunk 7]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+
+### 升级操作示例
+
+启动 dm-master/dm-worker 前确保废弃的配置信息已经删除，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

File: v3.0/reference/tools/data-migration/dm-upgrade.md [hunk 8]
```diff
@@ -20,6 +20,30 @@ aliases: ['/docs-cn/tools/dm/dm-upgrade/']
...
+
+### 升级操作示例
+
+启动 dm-master/dm-worker 前确保废弃的配置信息已经删除，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

File: v2.1/reference/tools/data-migration/dm-upgrade.md [hunk 9]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+可能遗留的废弃配置:
+
+- dm-worker.toml 中的 `meta-file`
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `dm-worker.toml` 中的 `meta-file`
```

File: v2.1/reference/tools/data-migration/dm-upgrade.md [hunk 10]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+- task.yaml 中的 `mysql-instances` 中的 `server-id`
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `task.yaml` 中的 `mysql-instances` 中的 `server-id`
```

File: v3.0/reference/tools/data-migration/dm-upgrade.md [hunk 11]
```diff
@@ -20,6 +20,30 @@ aliases: ['/docs-cn/tools/dm/dm-upgrade/']
...
+可能遗留的废弃配置:
+
+- dm-worker.toml 中的 `meta-file`
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `dm-worker.toml` 中的 `meta-file`
```

File: v3.0/reference/tools/data-migration/dm-upgrade.md [hunk 12]
```diff
@@ -20,6 +20,30 @@ aliases: ['/docs-cn/tools/dm/dm-upgrade/']
...
+- task.yaml 中的 `mysql-instances` 中的 `server-id`
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `task.yaml` 中的 `mysql-instances` 中的 `server-id`
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 13]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+可能遗留的废弃配置:
+
+- dm-worker.toml 中的 `meta-file`
```
Comment (lilin90) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `dm-worker.toml` 中的 `meta-file`
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 14]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+- task.yaml 中的 `mysql-instances` 中的 `server-id`
```
Comment (lilin90) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- `task.yaml` 中的 `mysql-instances` 中的 `server-id`
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 15]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
+
+### 升级操作示例
+
+启动 DM-master 或 DM-worker 前, 必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

File: v2.1/reference/tools/data-migration/dm-upgrade.md [hunk 16]
```diff
@@ -19,6 +19,30 @@ category: reference
...
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
+
+### 升级操作示例
+
+启动 DM-master 或 DM-worker 前, 必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

File: v3.0/reference/tools/data-migration/dm-upgrade.md [hunk 17]
```diff
@@ -20,6 +20,30 @@ aliases: ['/docs-cn/tools/dm/dm-upgrade/']
...
+Go Version: go version go1.11.2 linux/amd64
+```
+
+### 主要变更
+
+从此版本开始，将对所有的配置进行严格检查，遇到不识别的配置会报错，以确保用户始终准确地了解自己的配置。
+
+### 升级操作示例
+
+启动 DM-master 或 DM-worker 前,必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
启动 DM-master 或 DM-worker 前，必须确保已经删除废弃的配置信息，且没有多余的配置项，否则会启动失败。可根据失败信息删除多余的配置。
```

**Comment by lilin90** [comment]
@3pointer Please address comments. Thanks!

**Comment by 3pointer** [comment]
> @3pointer Please address comments. Thanks!

Done

**Pull request merged**